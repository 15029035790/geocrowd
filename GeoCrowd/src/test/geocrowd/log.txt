
##################
Budget = 1792
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_B    	MAX_COVER_ADAPT_T    	
0.100000             	40                   	40                   	40                   	
0.200000             	40                   	40                   	40                   	
0.300000             	40                   	40                   	40                   	
0.400000             	40                   	40                   	40                   	
0.500000             	40                   	40                   	40                   	

##################
Budget = 1792
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_B    	MAX_COVER_ADAPT_T    	
0.100000             	680                  	680                  	680                  	
0.200000             	680                  	678                  	680                  	
0.300000             	680                  	677                  	678                  	
0.400000             	680                  	677                  	680                  	
0.500000             	680                  	675                  	679                  	

##################
Budget = 224
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_B    	MAX_COVER_ADAPT_T    	
0.100000             	601                  	680                  	680                  	
0.200000             	601                  	677                  	680                  	
0.300000             	601                  	678                  	679                  	
0.400000             	601                  	677                  	678                  	
0.500000             	601                  	679                  	679                  	

##################
Budget = 896
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_B    	MAX_COVER_ADAPT_T    	
0.100000             	676                  	680                  	679                  	
0.200000             	676                  	677                  	679                  	
0.300000             	676                  	678                  	678                  	
0.400000             	676                  	677                  	679                  	
0.500000             	676                  	676                  	677                  	


##################Overloading
Budget = 336
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_MO   	
0.100000             	505                  	505                  	
MAX_COVER_BASIC      	MAX_COVER_BASIC_MO   	
0.100000             	5                    	2                    	


##################Overloading
Budget = 336
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_MO   	
0.100000             	684                  	559                  	
MAX_COVER_BASIC      	MAX_COVER_BASIC_MO   	
0.100000             	6                    	2                    	


##################Overloading
Budget = 336
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_MO   	
0.100000             	505                  	505                  	
MAX_COVER_BASIC      	MAX_COVER_BASIC_MO   	
0.100000             	5                    	2                    	


##################Overloading
Budget = 336
#Covered Tasks
                     	
0.100000             	

0.100000             	


##################Overloading
Budget = 336
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_MO   	MAX_COVER_BASIC_W_MO 	
0.100000             	505                  	506                  	274                  	
MAX_COVER_BASIC      	MAX_COVER_BASIC_MO   	MAX_COVER_BASIC_W_MO 	
0.100000             	5                    	2                    	2                    	


##################Overloading
Budget = 336
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_MO   	MAX_COVER_BASIC_W_MO 	
0.100000             	1003                 	988                  	644                  	
MAX_COVER_BASIC      	MAX_COVER_BASIC_MO   	MAX_COVER_BASIC_W_MO 	
0.100000             	5                    	1                    	2                    	

##################Local varying alpha
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.100000             	6870                 	5851                 	0                    	0                    	

##################Local varying alpha
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_T    	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.100000             	6870                 	9427                 	5851                 	0                    	0                    	

##################Local varying budget
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_T    	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
28                   	5368                 	5360                 	4824                 	1627                 	1627                 	
56                   	6501                 	6482                 	5871                 	882                  	882                  	
112                  	7605                 	7613                 	6503                 	1371                 	1371                 	
224                  	8344                 	8341                 	7592                 	1536                 	1536                 	
448                  	8827                 	8810                 	8087                 	1887                 	1887                 	
896                  	8992                 	8994                 	8572                 	2016                 	2016                 	
1792                 	9002                 	9002                 	8917                 	2128                 	2128                 	
3586                 	9002                 	9002                 	9002                 	2199                 	2199                 	

##################Local varying alpha
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_T    	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.100000             	6870                 	9455                 	5851                 	0                    	0                    	

##################Local varying alpha
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_T    	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.100000             	13385                	15167                	12947                	0                    	0                    	

##################Local varying alpha
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_T    	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.100000             	29282                	31113                	26294                	0                    	0                    	

##################Local varying alpha
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_T    	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.100000             	13605                	13625                	12851                	0                    	0                    	

##################Local varying alpha
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_T    	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.100000             	10121                	12874                	8900                 	0                    	0                    	

##################Local varying alpha
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_T    	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.100000             	10093                	12970                	9760                 	0                    	0                    	

##################Local varying budget
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_T    	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
28                   	5368                 	5360                 	4824                 	1576                 	1576                 	
56                   	6501                 	6482                 	5871                 	882                  	882                  	
112                  	7605                 	7613                 	6503                 	1371                 	1371                 	

##################Local varying budget
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_T    	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
28                   	4931                 	4909                 	4088                 	351                  	351                  	

##################Local varying delta
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_T    	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
1                    	117                  	117                  	114                  	0                    	0                    	
2                    	143                  	147                  	138                  	114                  	114                  	
3                    	161                  	164                  	157                  	138                  	138                  	
4                    	170                  	188                  	179                  	157                  	157                  	
5                    	186                  	193                  	186                  	179                  	179                  	
6                    	191                  	199                  	183                  	186                  	186                  	
7                    	193                  	205                  	200                  	183                  	183                  	
8                    	201                  	216                  	200                  	200                  	200                  	
9                    	206                  	212                  	208                  	200                  	200                  	
10                   	208                  	221                  	211                  	208                  	208                  	

##################Local varying delta
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_T    	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
1                    	3271                 	3271                 	3263                 	0                    	0                    	
2                    	3891                 	3893                 	3845                 	3263                 	3263                 	
3                    	4444                 	4973                 	4442                 	3845                 	3845                 	
4                    	5003                 	5671                 	5202                 	4442                 	4442                 	
5                    	5865                 	6260                 	5833                 	5202                 	5202                 	
6                    	6352                 	6705                 	6352                 	5833                 	5833                 	
7                    	6730                 	7160                 	6821                 	6352                 	6352                 	
8                    	6999                 	7441                 	7075                 	6821                 	6821                 	
9                    	7244                 	7811                 	7244                 	7075                 	7075                 	
10                   	7500                 	8023                 	7487                 	7244                 	7244                 	


##################Overloading
Budget = 336
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_S    	
0.100000             	13356                	7107                 	
MAX_COVER_BASIC      	MAX_COVER_BASIC_S    	
0.100000             	7                    	25    

##################Local varying alpha
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_T    	FixedOff             	DynamicOff           	
0.100000             	71.678629            	69.381998            	0.000000             	0.000000             	
0.100000             	1654.000000          	2194.000000          	0.000000             	0.000000             	
               	

##################Runtime
Budget = 56
#Covered Tasks
                     	MAX_COVER_ADAPT_T    	FixedOff             	DynamicOff           	
0.100000             	37.463628            	0.000000             	0.000000             	#Elapsed time

0.100000             	869.000000           	0.000000             	0.000000             	

##################Runtime
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_T    	FixedOff             	DynamicOff           	
0.100000             	38.175207            	40.566618            	0.000000             	0.000000             	
#Elapsed time

0.100000             	870.000000           	812.000000           	0.000000             	0.000000             	

##################Runtime
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_T    	FixedOff             	DynamicOff           	
0.100000             	72.501522            	74.305109            	0.000000             	0.000000             	
#Elapsed time

0.100000             	1733.000000          	2230.000000          	0.000000             	0.000000             	

##################Runtime
Budget = 56
#Covered Tasks
                     	MAX_COVER_ADAPT_T    	FixedOff             	DynamicOff           	
0.100000             	106.509213           	0.000000             	0.000000             	
#Elapsed time

0.100000             	3931.000000          	0.000000             	0.000000             	

##################Runtime
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_T    	FixedOff             	DynamicOff           	
0.100000             	131.380069           	138.650833           	0.000000             	0.000000             	
#Elapsed time

0.100000             	4035.000000          	5054.000000          	0.000000             	0.000000             	

##################Offline varying budget

                     	true                 	false                	
28                   	0.000000             	0.000000             	

##################Offline varying budget

                     	true                 	false                	
28                   	155.307734           	158.959663           	

##################Local varying radius
Budget = 28
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_B    	MAX_COVER_ADAPT_T    	FixedOff             	DynamicOff           	
1.000000             	26.167701            	29.229917            	27.336939            	0.000000             	0.000000             	
5.000000             	80.998728            	79.707554            	80.498666            	27.336939            	27.336939            	
10.000000            	147.409770           	147.824783           	148.071704           	80.498666            	80.498666            	
15.000000            	204.075285           	203.808845           	203.875710           	148.071704           	148.071704           	
20.000000            	266.523219           	267.122322           	266.058529           	203.875710           	203.875710           	

##################Local varying radius
Budget = 14
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_B    	MAX_COVER_ADAPT_T    	FixedOff             	DynamicOff           	
1.000000             	20.476839            	21.088609            	20.621124            	0.000000             	0.000000             	
10.000000            	138.365462           	143.590154           	145.770543           	20.621124            	20.621124            	
20.000000            	262.239202           	265.616441           	264.661171           	145.770543           	145.770543           	         	

##################Local varying radius
Budget = 14
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_B    	MAX_COVER_ADAPT_T        	
1.000000             	17.957349            	20.672850            	18.794732           	
10.000000            	133.135168           	142.153269           	136.104292       	
20.000000            	258.262549           	260.621891           	260.360468       	

##################Local varying radius
Budget = 28
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_B    	MAX_COVER_ADAPT_T  	
1.000000             	25.083136            	25.234001            	25.832135         	
10.000000            	137.644450           	138.737692           	145.587784   	
20.000000            	255.840336           	256.554856           	260.900838       	

##################Local varying radius
Budget = 28
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_B    	MAX_COVER_ADAPT_T    	FixedOff             	DynamicOff           	
1.000000             	25.083136            	22.919455            	24.292983            	0.000000             	0.000000             	
10.000000            	137.644450           	143.194115           	137.494360           	24.292983            	24.292983            	
20.000000            	255.840336           	258.786809           	252.589384           	137.494360           	137.494360           	

##################Local varying radius
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_B    	MAX_COVER_ADAPT_T    	FixedOff             	DynamicOff           	
1.000000             	49.187455            	50.151110            	46.966536            	0.000000             	0.000000             	
2.000000             	88.310907            	90.631415            	88.801081            	46.966536            	46.966536            	
3.000000             	131.828853           	131.893944           	119.947762           	88.801081            	88.801081            	

##################Local varying radius
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_B    	MAX_COVER_ADAPT_T         	
1.000000             	118.197519           	124.801053           	123.731012         	
5.000000             	407.691685           	409.767953           	430.094696          	
10.000000            	686.445072           	690.214737           	701.556811         	

##################Local varying radius
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_B    	MAX_COVER_ADAPT_T      	
1.000000             	5.332681             	4.685258             	4.861441     	
5.000000             	13.503185            	13.263571            	13.319663        	
10.000000            	17.534194            	17.243343            	16.284493        	

##################Local varying radius
Budget = 28
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_ADAPT_B    	MAX_COVER_ADAPT_T    	FixedOff             	DynamicOff           	
1.000000             	60.677091            	63.353169            	61.834728            	0.000000             	0.000000             	
10.000000            	364.536608           	367.583304           	362.418593           	61.834728            	61.834728            	
100.000000           	1650.400443          	1651.161934          	1653.048293          	362.418593           	362.418593           	
1000.000000          	1967.942778          	1967.942778          	1967.942778          	1653.048293          	1653.048293          	
