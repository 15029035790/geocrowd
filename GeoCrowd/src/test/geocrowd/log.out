
##################Local varying alpha
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_T    	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.200000             	791                  	715                  	790                  	793                  	958                  	

##################Local varying alpha
Budget = 112
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_T    	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.200000             	6711                 	7102                 	6711                 	7668                 	8450                 	

##################Local varying alpha
Budget = 112
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_T    	FixedOff             	DynamicOff           	
0.200000             	6711                 	6812                 	7668                 	8450                 	

##################Local varying alpha
Budget = 112
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_T    	MAX_COVER_BASIC_T2   	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.200000             	6711                 	7102                 	6763                 	6729                 	7668                 	8450                 	

##################Local varying alpha
Budget = 112
#Covered Tasks
                     	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.200000             	4456                 	7668                 	8450                 	

##################Local varying alpha
Budget = 112
#Covered Tasks
                     	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.200000             	4963                 	7668                 	8450                 	

##################Local varying alpha
Budget = 112
#Covered Tasks
                     	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.200000             	4386                 	7668                 	8450                 	

##################Local varying alpha
Budget = 112
#Covered Tasks
                     	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.200000             	4760                 	7668                 	8450                 	

##################Local varying alpha
Budget = 224
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_T    	FixedOff             	DynamicOff           	
0.200000             	8539                 	9208                 	10040                	10927                	

##################Local varying alpha
Budget = 112
#Covered Tasks
                     	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.200000             	4760                 	7668                 	8450                 	

##################Local varying alpha
Budget = 112
#Covered Tasks
                     	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.200000             	6711                 	7668                 	8450                 	

##################Local varying alpha
Budget = 112
#Covered Tasks
                     	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.200000             	6711                 	7668                 	8450                 	

##################Local varying alpha
Budget = 112
#Covered Tasks
                     	MAX_COVER_BASIC_T2   	MAX_COVER_BASIC_S2   	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.200000             	6763                 	6738                 	6715                 	7668                 	8450                 	

##################Local varying alpha
Budget = 112
#Covered Tasks
                     	MAX_COVER_BASIC_T2   	MAX_COVER_BASIC_S2   	MAX_COVER_BASIC_S    	FixedOff             	DynamicOff           	
0.200000             	6763                 	6692                 	4529                 	7668                 	8450                 	

##################Local varying alpha
Budget = 112
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_S2   	FixedOff             	DynamicOff           	
0.200000             	6711                 	6731                 	7668                 	8450                 	

##################Local varying alpha
Budget = 112
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_S2   	FixedOff             	DynamicOff           	
0.200000             	6711                 	6692                 	7668                 	8450                 	

##################Local varying alpha
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_S2   	FixedOff             	DynamicOff           	
0.200000             	4880                 	4808                 	5656                 	6333                 	

##################Local varying alpha
Budget = 56
#Covered Tasks
                     	MAX_COVER_BASIC      	MAX_COVER_BASIC_S2   	FixedOff             	DynamicOff           	
0.200000             	4880                 	4798                 	5656                 	6333                 	
